magnet/excitation-data/si-corrector-ids-ch.txt

```diff
@@ -8,13 +8,14 @@
 
 # EXCITATION DATA
 # ===============
--005.000  -3.0000e-04 +0.0000e+00
-+005.000  +3.0000e-04 +0.0000e+00
+-001.000  -1.6409e-04 +0.0000e+00
++001.000  +1.6409e-04 +0.0000e+00
```

Comment:
@xresende, isn't these values for half of the model?

Reply:
ops! they are indeed for half model. thanks. I'll fix it!

Reply:
done.